conf/modules/short_variant_calling.config

```diff
@@ -25,8 +25,8 @@ process {
     }
 
     withName: '.*:SHORT_VARIANT_CALLING:DEEPVARIANT' {
-
-        ext.prefix = { intervals ? "${meta.id}" + "_" + "${intervals.getSimpleName()}" + "_deepvariant" : "${meta.id}" + "_deepvariant"}
+        // Needed to not cause input name collision in bcftools concat
```

Comment:
I would remove this comment even if its good to see why its there first time you read it :)

Reply:
Sure